### DISCUSSION TITLE
Issue with apple login 'null is not an object [evaluating 't[e]')'

### DISCUSSION THREAD

**Discussion by Haris565** [Web3Auth/Web3Auth#707] [Help]
We are trying to integrate apple login via plug and play and sometime this error comes up and when you restart the app it works properly fine what could be the reason of this error ? We need to deploy the app and face rejection 3 times because of this and still not able to resolve the issue 
![IMG_5991](https://user-images.githubusercontent.com/64888353/194843478-f9b6286f-4231-47ad-90bb-035e2859d8c5.png)
![code](https://user-images.githubusercontent.com/64888353/194844834-345f4069-5884-4244-b9ce-0a474b2ce2e8.png)

**Comment by BboyStatix** [comment]
Hi @Haris565 I'm going to try and reproduce the issue but just from looking at the code alone I have a hunch that it may due to the fact that you are initialising web3auth every time the login function is called.

Could you try refactoring your code to make it more like our [QuickStart example](https://web3auth.io/docs/integration-builder?lang=react&chain=eth&customAuthentication=no&whitelabel=no&mfaParams=no&usingEmailPasswordless=no&rnWorkflowMode=expo&evmFramework=web3#step-0)?

**Comment by Lauman** [comment]
Hi.
Any ideas? It only happens to me on ios.

**Comment by chaitanyapotti** [comment]
Please provide info about the sdk version you're using

**Reply by Lauman** [reply]
"@web3auth/react-native-sdk": "~3.2.1"
"@toruslabs/react-native-web-browser": "~1.1.0"

**Comment by chaitanyapotti** [comment]


**Reply by Lauman** [reply]
Hello @chaitanyapotti.
What do you mean by call init?
Thanks.

**Comment by chaitanyapotti** [comment]
Please upgrade to the latest version.
We're unable to reproduce this issue.
Pls provide a link to your test flight app
